client/app/scripts/superdesk-archive/module.js

```diff
@@ -106,6 +106,79 @@ define([
         };
     }
 
+    ListLayoutController.$inject = ['preferences'];
+    function ListLayoutController(preferences) {
+        var MAX_VIEW = {max: true},
+            MIN_VIEW = {min: true},
+            ITEM_VIEW = {extended: true},
+            COMPACT_VIEW = {compact: true};
```

Comment:
We might want to use [Object.freeze()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) for these kind of objects.